apps/example-tailwind/src/app/app.routes.ts

```diff
@@ -33,15 +33,15 @@ const authRoutes: Route[] = [
             import('@myorg/app-example-auth').then(
                 (m) => m.PageAccountCreatedComponent,
             ),
-        resolve: { layout: setLayout(PageLayout.Center) },
+        resolve: { layout: setLayout(PageLayout.Cta) },
     },
     {
         path: 'account-verify/:token',
         loadComponent: () =>
             import('@myorg/app-example-auth').then(
                 (m) => m.PageAccountVerifyComponent,
             ),
-        resolve: { layout: setLayout(PageLayout.Center) },
+        resolve: { layout: setLayout(PageLayout.Cta) },
```

Comment:
why use cta? remove it please use the center
```suggestion
        resolve: { layout: setLayout(PageLayout.center) }
```